vars/pipeline2ATX.groovy

```diff
@@ -93,9 +94,11 @@ def getRawBuild(String jobName, int buildNumber) {
  */
 def getBuildAttributes(build) {
     def attributes = []
-    def buildUrl = build.absoluteUrl
-    def buildId = build.id
-    def buildAttributes = [BUILD_URL: buildUrl, BUILD_ID: buildId]
+    def buildAttributes = [PRODUCT_VERSION: "${PRODUCT_VERSION}".toString(), 
```

Comment:
I'm not sure whether we should force users to use our coverage structure. The attributes `PRODUCT_NAME`, `PRODUCT_VERSION` and `TEST_LEVEL` is no default definition of TG